docs/compilers/CSharp/Compiler Breaking Changes - post VS2017.md

```diff
@@ -130,4 +130,7 @@ Example:
 
 - Visual Studio 2017 version 15.8: https://github.com/dotnet/roslyn/issues/22455 C# compiler will now produce errors if there was an "in" or an "out" argument to an "__arglist" call. "out" was always allowed, and "in" was introduced in 15.5.
 - Visual Studio 2017 version 15.8: https://github.com/dotnet/roslyn/issues/26418 C# compiler will now produce errors on out variable declarations that have "ref" or "ref readonly" ref kinds. Example: M(out ref int x);
-- Visual Studio 2017 version 15.8: https://github.com/dotnet/roslyn/issues/27047 The C# compiler will now produce diagnostics for operators marked as obsolete when they are used as part of a tuple comparison.
\ No newline at end of file
+- Visual Studio 2017 version 15.8: https://github.com/dotnet/roslyn/issues/27047 The C# compiler will now produce diagnostics for operators marked as obsolete when they are used as part of a tuple comparison.
+
+- Visual Studio 2017 version 15.8: https://github.com/dotnet/roslyn/issues/26743 C# will now reject expressions such as `public int* M => &this.Bar[0];` if `Bar` is a fixed field of the containing type.
+- Visual Studio 2017 version 15.8: https://github.com/dotnet/roslyn/issues/26688 C# will now accept expressions such as `public int M => Bar[0];` if `Bar` is a fixed field of the containing type.
```

Comment:
Is this really a breaking change?

Reply:
Technically no. I'll remove it.

---
In reply to: [194576467](https://github.com/dotnet/roslyn/pull/27424#discussion_r194576467) [](ancestors = 194576467)